RecoEgamma/EgammaTools/src/PhotonIDHelper.cc

```diff
@@ -0,0 +1,61 @@
+#include "RecoEgamma/EgammaTools/interface/PhotonIDHelper.h"
+
+#include <iostream>
+
+PhotonIDHelper::PhotonIDHelper(const edm::ParameterSet  & iConfig,edm::ConsumesCollector && iC):
```

Comment:
considering identical implementation compared to ElectronIDHelper,
both can be derived from a template.

Reply:
Someday they may require more and different products from the event, so we expect the classes to eventually diverge from each other.

Reply:
Perhaps, although I was hoping that by that someday this really becomes a part of reco and not a helper that generates user floats for pat.